gtda/mapper/visualization.py

```diff
@@ -291,6 +300,88 @@ def plot_interactive_mapper_graph(
 
 
 class MapperInteractivePlotter:
+    """Plot Mapper graphs in a Jupyter session, with interactivity on pipeline
+    parameters.
+
+    Provides functionality to interactively update parameters from the cover,
+    clustering and graph construction steps defined in `pipeline`.
+    Interactivity is via a widget produced when calling :meth:`plot`. After
```

Comment:
Suggestion:

An interactive widget is produced when calling :meth:`plot`

Reply:
Thanks! Will do